.vscode/launch.json

```diff
@@ -67,7 +67,8 @@
                 "--extensionDevelopmentPath=${workspaceFolder}/../vscode-azureappservice",
                 "--extensionDevelopmentPath=${workspaceFolder}/../vscode-cosmosdb",
                 "--extensionDevelopmentPath=${workspaceFolder}/../vscode-azurecontainerapps",
-                "--extensionDevelopmentPath=${workspaceFolder}/../vscode-azurespringcloud"
+                "--extensionDevelopmentPath=${workspaceFolder}/../vscode-azurespringcloud",
+                "--extensionDevelopmentPath=${workspaceFolder}/../vscode-azurelogicapps"
```

Comment:
I think I've asked this before, but does VSCode get upset when you do debug launch, and one (or more) of these paths does not exist?